nucleus/cluster/cli/src/main/java/com/sun/enterprise/admin/cli/cluster/ChangeNodeMasterPasswordCommand.java

```diff
@@ -188,6 +189,22 @@ protected void createMasterPasswordFile() throws CommandException {
     }
 
 
+    protected void chmod(String args, File file) throws IOException {
```

Comment:
Glassfish depends on Java 7, doesn't it? There is a native support for Posix permissions in NIO2, no need to fiddle with a new process.